### DISCUSSION TITLE
Expose toRgba

### DISCUSSION THREAD

**Discussion by testerez** [mantinedev/mantine#4200] [Feature requests]
Could you please expose `toRgba` helper function so that I can use it?

**Comment by yverby** [comment]
This is not what you need?
https://mantine.dev/theming/functions/#rgba

**Comment by testerez** [comment]
Thanks for answering @yverby , `rgba` returns a string and is used to set color alpha. What I need is parsing a color and return `r` `g` `b` and `a` as numbers, which is exactly what `toRgba` does. Unfortunately, it's not exposed.

**Comment by testerez** [comment]
Btw, is there a reason to expose helpers like `rgba` only through the theme, even though they have no dependancy with the theme?
